test/runTests.sh

```diff
@@ -39,12 +39,7 @@ do
         ((num_pass+=1))
     else
         ((num_fail+=1))
-    fi 
-	echo "=================================="
-	echo "output"
-	echo "=================================="
-	cat $test/test.out
```

Comment:
This file is not guaranteed to be used for every test. This check can be done in check_result instead for specific test.